### PR TITLE
Log commands

### LINKED ISSUE #40: Log when a command is initializing and when it interrupts

### PR DESCRIPTION
**Pull request by EliyaFishman**
This PR adds logging when a command start, end or gets interrupted
Fixes #40

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -60,6 +60,11 @@ public void robotInit() {
         // autoChooser.setDefaultOption(name, object);
         // autoChooser.addOption(name, object);
 
+        //set up command logging
+        CommandScheduler.getInstance().onCommandInitialize((command) -> DriverStationLogger.logToDS("Starting to run " + command.getName().toLowerCase()));
+        CommandScheduler.getInstance().onCommandInterrupt((command) -> DriverStationLogger.logToDS("Interrupting " + command.getName().toLowerCase()));
+        CommandScheduler.getInstance().onCommandFinish((command) -> DriverStationLogger.logToDS(command.getName() + " is finished"));
+
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/Robot.java:67]: //set up command logging
        CommandScheduler.getInstance().onCommandInitialize((command) -> DriverStationLogger.logToDS("Starting to run " + command.getName().toLowerCase()));
        CommandScheduler.getInstance().onCommandInterrupt((command) -> DriverStationLogger.logToDS("Interrupting " + command.getName().toLowerCase()));
        CommandScheduler.getInstance().onCommandFinish((command) -> DriverStationLogger.logToDS(command.getName() + " is finished"));
did you ment to put those in periodic? not robot init... 
also you dont need (command), use:
command -> ...

**Review by HilelAmiel — CHANGES_REQUESTED** [review]
one thing

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (EliyaFishman) [reply src/main/java/frc/robot/Robot.java:67]: I did not mean to put it in periodic. The command scheduler calls those lambdas every time a command start/interrupt/finishes.
see [WPILib example](https://github.com/wpilibsuite/allwpilib/tree/master/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/schedulereventlogging)
and [the docs](https://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj2/command/CommandScheduler.html#onCommandInitialize(java.util.function.Consumer))

**Comment by EliyaFishman** [comment]
[Log commands to DS](https://trello.com/c/fjIePU4g/132-log-commands-to-ds)

**Review by HilelAmiel — APPROVED** [review]

**Pull request merged**